client/src/components/InitialPromptCreationScreen/InitialPromptCreationScreen.jsx

```diff
@@ -8,18 +8,54 @@ class InitialPromptCreationScreen extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      noun: '',
+      adjective1: '',
+      adjective2: '',
       error: null,
     };
+
+    this.onSubmitPromptsButtonClicked = this.onSubmitPromptsButtonClicked.bind(this);
     this.updateGameState = this.updateGameState.bind(this);
+    this.onNounChange = this.onNounChange.bind(this);
+    this.onAdjective1Change = this.onAdjective1Change.bind(this);
+    this.onAdjective2Change = this.onAdjective2Change.bind(this);
+  }
+
+  componentDidMount() {
+    const timerId = setInterval(this.updateGameState, 3000);
```

Comment:
Do we want to listen starting at component mount? Or would it make more sense to do it once the current player submits their prompt since we need to wait for that anyway?

Reply:
Also - can you use the helper I added that already does that for you? It's here https://github.com/danbarragan/drawydraw/blob/master/client/src/utils/updateGameState.js#L3

Reply:
you can use it like this https://github.com/danbarragan/drawydraw/blob/8f5af064ff519c0375f6d8a146c816c1c1532f2c/client/src/components/VotingScreen/VotingScreen.jsx#L20-L30

Reply:
Ah! that makes sense, thanks.